### ISSUE TITLE
Feature Request: Send removed items

### ISSUE THREAD

**Issue by Raandom** [CitizensDev/Citizens#818]
If you remove items etc at the begin or at the end from the quest from the player, you can define a trader that gets thoose items.

Would be nice.

**Comment by fjutte** [comment]
Yes it would; for now the solution is creating another quest B require completion of quest A. Quest B will then "give back" the items that were removed in quest A.
